pages/api/dummyData.js

```diff
@@ -0,0 +1,108 @@
+// Uses Structure from dictionary API, but adds study data for later
+export const dummyData = [
+  {
+    isDictionaryEntry: false,
+    slug: "家庭",
+    japanese: [
+      {
+        word: "家庭",
+        reading: "かてい",
+      },
+    ],
```

Comment:
Why is this an array? Do you plan to contain more Japanese references per word? ^^

Reply:
I'm not sure why they did that. For the actual translations I used the same structure as provided by the external API (https://jisho.org/api/v1/search/words?keyword=dictionary) because I might want to use some components with the external (dictionary) and internal (user-created) word lists and then it's probably easier this way.

Reply:
Then if you plan to actually query the data from the API, I recommend to have a `parseData` function somewhere, that shapes it in the format you need, removing unnecessary information and naming it like you plan to use it inside your app, e.g.:

```js
function parseData(data) {
  return data.map((item) => {
    return {
      japaneseDefinition: item.japanese[0].word,
      japaneseReading: item.japanese[0].reading,
      englishDefinitions: item.english_definitions,
      // rest of what you need
    };
  });
}
```